README.md

```diff
@@ -1,98 +1,79 @@
-# Open Platform for AI (PAI) ![alt text][logo]
+# Open Platform for AI (OpenPAI) ![alt text][logo]
 
 [logo]: ./pailogo.jpg "OpenPAI"
 
 [![Build Status](https://travis-ci.org/Microsoft/pai.svg?branch=master)](https://travis-ci.org/Microsoft/pai)
 [![Coverage Status](https://coveralls.io/repos/github/Microsoft/pai/badge.svg?branch=master)](https://coveralls.io/github/Microsoft/pai?branch=master)
 
+OpenPAI is an open source platform that provides complete AI model training and resource management capabilities, it is easy to extend and supports on-premise, cloud and hybrid environments in various scale. 
 
-## Introduction
+# Table of Contents
+1. [When to consider OpenPAI](#when-to-consider-openpai)
+2. [Why choose OpenPAI](#why-choose-openpai)
+3. [How to deploy](#how-to-deploy)
+4. [How to use](#how-to-use)
+5. [Resources](#resources)
+6. [Get Involved](#get-involved)
+7. [How to contribute](#how-to-contribute)
 
-Platform for AI (PAI) is a platform for cluster management and resource scheduling.
-The platform incorporates the mature design that has a proven track record in Microsoft's large scale production environment.
+## When to consider OpenPAI
+1. When your organization need a better utilization of AI computing resources (GPU/CPU etc.). On demand allocates resources. 
```

Comment:
>When your organization need a better utilization of AI computing resources (GPU/CPU etc.). On demand allocates resources.  [](start = 3, length = 122)

When your organization nedd to share powerful AI computing resources (GPU/FGPA farm, etc.) among teams.